compiler/pragmas.nim

```diff
@@ -175,9 +175,15 @@ proc processImportCompilerProc(c: PContext; s: PSym, extname: string, info: TLin
   incl(s.loc.flags, lfImportCompilerProc)
 
 proc processImportCpp(c: PContext; s: PSym, extname: string, info: TLineInfo) =
+  var extname = extname
+  let isFreeFunction = extname.startsWith "!"
```

Comment:
add some comments here, clearifying the pattern of free function

Reply:
done